### DISCUSSION TITLE
Video not showing for Portrait Videos

### DISCUSSION THREAD

**Discussion by ghost** [photoprism/photoprism#4048] [Q&A]
Hello,

I have just recently started to use PhotoPrism. It's installed on a Debian LXC container on Proxmox. I am accessing the uploaded images/videos on my Windows machine using Edge browser (everything is up to date). I have noticed that when I play the Landscape videos, it works fine.... but for Portrait vidoes, I see the thumbnail and I can hear the audio... but there is no video, just black screen. When I try to access the same video using photoprism on my phone (Chrome and Edge browser), the video shows up in potrait mode but this time it's rotated (stretched). When I try to directly play the video file using Media Player or VLC, it shows up fine. 

I have tried the video taken from Google Pixel 6 phone and Iphone 13, both have the same issue.... Below is the file details for one of the video that has this issue...

![Screenshot 2024-02-01 130740](https://github.com/photoprism/photoprism/assets/19824336/70dbe700-ccf5-4e7b-b010-e824f962edb2)

I would really appreciate if someone can help me here regarding this issue....

**Comment by m2gChristian** [comment]
I have exactly same issue (but I did not tested on my Pixel Phone). Only tested in my Edge Browser on Windows 11.
The portrait videos do not show video (only black) but sound and time is ok, and landscape video files working fine.

I installed PhotoPrism on Unraid 6.12.6 through community apps (docker).

**Comment by m2gChristian** [comment]
Hi, I just tried to install Google Chrome additional to Edge and there I do not have this issue on the same machine.

**Comment by graciousgrey** [comment]
Thank you very much for reporting!

@isaranghi Can you confirm that the problem only occurs with Edge and not with a different browser?

**Reply by ghost** [reply]
Yes... I tried it on Google Chrome and Portrait videos are loading fine... I guess it's Microsoft Edge issue....

Thanks....

**Reply by graciousgrey** [reply]
I just tested this (on browserstack) with Windows 11 and Edge 121, I was able to play videos in portrait and landscape mode without any problems. So it could just be a problem for some files or it could depend on some browser settings or extensions.
